provision/api/endpoint.go

```diff
@@ -35,3 +35,24 @@ func doProvision(svc provision.Service) endpoint.Endpoint {
 
 	}
 }
+
+func doCert(svc provision.Service) endpoint.Endpoint {
```

Comment:
What does this function do? Creates Cert? If yes, consider calling it `createCert()`. Or just `Cert()`.